cv/pothole_detection/launch/pothole_detection_server.launch

```diff
@@ -0,0 +1,19 @@
+<launch>
+    <!-- Runs CV pipeline in python3 communicating with UDP pipeline -->
+    <node
+        name="pothole_detection_model_inference"
+        pkg="pothole_detection"
+        type="pothole_detection_model_inference.py"
+        output="screen"
+    >
+        <remap from="image" to="/zed/zed_node/rgb/image_rect_color"/>
```

Comment:
This should be `left` instead of `rgb` since we don't have a `rgb` topic (in sim). Again, don't know what the real camera does but I anticipate it is similar

Reply:
rgb in real zed wrapper corresponds to depth map. This should be like this, the sim is missing the topic. I think it maps to left camera